Source/FieldSolver/SpectralSolver/SpectralAlgorithms/PsatdAlgorithm.H

```diff
@@ -1,23 +1,31 @@
 #ifndef WARPX_PSATD_ALGORITHM_H_
 #define WARPX_PSATD_ALGORITHM_H_
 
+//#include <SpectralKSpace.H>
+//#include <SpectralFieldData.H>
```

Comment:
Is there a reason to comment these lines (and define the `SpectralCoefficients` type below)?
If not could you uncomment them? (and remove the definition of `SpectralCoefficients` below)?

Reply:
There was an update in the dev branch sometime in between. When I did the merge, I found that the Source/FieldSolver/SpectralSolver/SpectralAlgorithms/PsatdAlgorithm.H  in the dev branch did not have the include statements and it had the defined SpectralCoefficients type. 
Line 21 of the PsatdAlgorithm.H file in the 'dev' branch has the same. 

Please let me know if this still requires change. I have deleted the commented #include lines to avoid confusion.

Reply:
Yes, that makes sense: these include statements are now in `SpectralBaseAlgorithm.H`. So there is no need to have them in `PsatdAlgorithm.H`.